pros/common/sentry.py

```diff
@@ -24,7 +26,8 @@ def prompt_to_send(event: Dict[str, Any], hint: Optional[Dict[str, Any]]) -> Opt
         if 'extra' in event and not event['extra'].get('sentry', True):
             ui.logger(__name__).debug('Not sending candidate event because event was tagged with extra.sentry = False')
             return
-        if 'exc_info' in hint and not getattr(hint['exc_info'][1], 'sentry', True):
+        if 'exc_info' in hint and (not getattr(hint['exc_info'][1], 'sentry', True) or
+                                   any(hint['exc_info'][0] is e for e in SUPPRESSED_EXCEPTIONS)):
```

Comment:
Are you sure you want to be using `is` here?

Reply:
No, but it works. `exc_info[0]` is the type of the exception. I'm not sure how it'll work for subclasses, but none of the exceptions we suppress have subclasses.

Reply:
If you're doing type checking, why not `isinstance`?